File-Formats/Presentation/Working-with-Charts.md

```diff
@@ -702,12 +702,80 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Chart to Image conversion
 
+You can convert the chart in Word document as image using the [SaveAsImage](https://help.syncfusion.com/cr/file-formats/Syncfusion.OfficeChartToImageConverter.ChartToImageConverter.html#Syncfusion_OfficeChartToImageConverter_ChartToImageConverter_SaveAsImage_Syncfusion_OfficeChart_IOfficeChart_System_IO_Stream_) method in [ChartToImageConverter](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.WordDocument.html#Syncfusion_DocIO_DLS_WordDocument_ChartToImageConverter).
```

Comment:
This is presentation control, but the contents are related to DocIO.

Use the below sentence
>>>>

To convert chart in PowerPoint slide as an image, refer the below dependencies in your application.
>>>>>>